d2-int-test/build.gradle

```diff
@@ -13,9 +13,11 @@ dependencies {
   compile externalDependency.zookeeper
```

Comment:
Pegasus follows older code style such as bracket on new line. See here: https://iwww.corp.linkedin.com/wiki/cf/display/ENGS/Service+Infrastructure+Java+Coding+Conventions

all files should have the copyright statement at the top ... which reminds me I need to include in my PR too

Reply:
Good to know! Thank you!